README.md

```diff
@@ -47,6 +47,13 @@ Examples:
 
 * Lists all running mosaic chain containers:
   * `./mosaic list`
+  
+  ## Available chains
+  ```
+  1406, 1407 - Testnet auxiliary chains.
+  1414 - Production auxiliary chain.
+  ```
```

Comment:
please remove the leading spaces as this weirdly indents the paragraph (see in preview; including the header)

Reply:
@schemar , It was done to make sub-section under `##Mosaic Chains`.